apluslms_roman/cli.py

```diff
@@ -305,13 +326,23 @@ def build_action(context):
     config = get_config(context)
     engine = get_engine(context)
 
+    builder = engine.create_builder(config)
+
+    # TODO: option to print more?
+    if (context.parser.parse_args().steps):
```

Comment:
No need to re-parse arguments. `context.args.steps`.

So, if steps can be listed with option `--list-steps` or `--steps=?`, then this would become, `if context.args.list_steps or (context.args.steps and context.args.steps[0] == '?':`.

In addition, TODO is implemented here?

Reply:
what I meant by "option to print more" is option to print more data of the steps, currently it's just the img.

Reply:
context.args.list_steps doesn't work. list-steps currently:
`parser.add_argument('-ls', '--list-steps', action='store_true')`

Reply:
at least in zsh steps=? doesn't work, the ? needs to be quoted.